zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -1320,6 +1320,43 @@ paths:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
 
+  /element-instances/{elementInstanceKey}/variables:
+    post:
+      tags:
+        - Element instance
+      summary: Updates element instance variables
+      description: |
+        Updates all the variables of a particular scope (e.g. process instance, flow element instance) with the given variable data.
+      parameters:
+        - name: elementInstanceKey
+          in: path
+          required: true
+          description: The key of the element instance to which to update the variables.
```

Comment:
🔧 Let's try to provide more hints on where this can be fetched from as a user. The gRPC API gives more details here that we should reuse.

> can be the process instance key (as obtained during instance creation), or a given element, such as a service task (see elementInstanceKey on the job message)